index.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+<link href="https://fonts.googleapis.com/css2?family=Elsie&display=swap" rel="stylesheet">
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Sarah Graywood Portfolio</title>
+    <link href="index.css" rel="stylesheet" type="text/css" />
+
+   <link rel="apple-touch-icon" sizes="57x57" href="/apple-icon-57x57.png">
+<link rel="apple-touch-icon" sizes="60x60" href="/apple-icon-60x60.png">
+<link rel="apple-touch-icon" sizes="72x72" href="/apple-icon-72x72.png">
+<link rel="apple-touch-icon" sizes="76x76" href="/apple-icon-76x76.png">
+<link rel="apple-touch-icon" sizes="114x114" href="/apple-icon-114x114.png">
+<link rel="apple-touch-icon" sizes="120x120" href="/apple-icon-120x120.png">
+<link rel="apple-touch-icon" sizes="144x144" href="/apple-icon-144x144.png">
+<link rel="apple-touch-icon" sizes="152x152" href="/apple-icon-152x152.png">
+<link rel="apple-touch-icon" sizes="180x180" href="/apple-icon-180x180.png">
+<link rel="icon" type="image/png" sizes="192x192"  href="/android-icon-192x192.png">
+<link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png">
+<link rel="icon" type="image/png" sizes="96x96" href="/favicon-96x96.png">
+<link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png">
+<link rel="manifest" href="/manifest.json">
+<meta name="msapplication-TileColor" content="#ffffff">
+<meta name="msapplication-TileImage" content="/ms-icon-144x144.png">
+<meta name="theme-color" content="#ffffff">
+</head>
+
+<body>
+        <div id="name"> 
+            Sarah<br>Graywood 
+    </div>
+
+    <p>
+        <img class=profile src="https://i.postimg.cc/3NV8Rjk0/Pics-Art-01-04-07-23-51.jpg"
+            class="profile" />
+    </p>
+    <hr />          
+      <br />
+      <button><a href="index.html">Home</a></button>
+      <button><a href="projects.html">Projects</a></button>
+      <button><a href="about.html">About</a></button>
+<button><a href="contact.html">Contact</a></button>
+            <br>
+    <div id="container">
+        <h2 class="projects">
+
+                <a href="https://github.com/smgraywood" target="_blank"></a>
+            </span>
+        </h2>
+    </div>
+    <br />
+    <div id=containered>
+    <div id="container">
+        <div class=list>
+            <ul>
+                <li class="github">
+                    <a href="https://github.com/smgraywood" target="_blank">
+                        <img alt="gitHub" src="https://image.flaticon.com/icons/png/512/25/25231.png" />
```

Comment:
I love all the social media link ideas! I want to add the same social media to my page!